Source/JavaScriptCore/Configurations/Base.xcconfig

```diff
@@ -108,7 +108,9 @@ CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
 PREBINDING = NO;
 WARNING_CFLAGS = -Wall -Wextra -Wcast-qual -Wchar-subscripts -Wconditional-uninitialized -Wextra-tokens -Wformat=2 -Winit-self -Wmissing-format-attribute -Wmissing-noreturn -Wpacked -Wpointer-arith -Wredundant-decls -Wundef -Wwrite-strings -Wexit-time-destructors -Wglobal-constructors -Wtautological-compare -Wimplicit-fallthrough -Wvla -Wliteral-conversion -Wthread-safety -Wno-profile-instr-out-of-date -Wno-profile-instr-unprofiled;
 
-HEADER_SEARCH_PATHS = . "${BUILT_PRODUCTS_DIR}/usr/local/include" $(HEADER_SEARCH_PATHS);
+HEADER_SEARCH_PATHS = . "$(BUILT_PRODUCTS_DIR)$(WK_STATICLIB_INSTALL_PREFIX)/include" $(HEADER_SEARCH_PATHS);
+SYSTEM_HEADER_SEARCH_PATHS = $(SDK_DIR)$(WK_STATICLIB_INSTALL_PREFIX)/include $(inherited);
```

Comment:
Is there a difference between `SDK_DIR` and `SDKROOT`? I've seen the latter used much more often, but I don't know if there's a semantic difference between the two.

Reply:
I think they behave the same, although SDKROOT can be set to a name (e.g. `iphoneos`) and it'll be transparently replaced with the path to the SDK directory. I think I reached for SDK_ROOT because it's used in a few similar places:

```
> git grep -w SDK_DIR main '**/*.xcconfig' 
main:Source/JavaScriptCore/Configurations/JavaScriptCore.xcconfig:WTF_ARCHIVE[config=Production] = $(SDK_DIR)$(WK_ALTERNATE_WEBKIT_SDK_PATH)/usr/local/lib/libWTF.a;
main:Source/WTF/Configurations/Base.xcconfig:SYSTEM_HEADER_SEARCH_PATHS = $(SDK_DIR)$(WTF_INSTALL_PATH_PREFIX)/usr/local/include $(inherited);
main:Source/WTF/Configurations/Base.xcconfig:LIBRARY_SEARCH_PATHS = $(SDK_DIR)$(WTF_INSTALL_PATH_PREFIX)/usr/local/lib $(inherited);
```